studio/src/deskStructure.ts

```diff
@@ -195,7 +196,7 @@ export default () =>
               }).icon(MenuIcon),
             ])
             .title("Navigation")
-        ),
+        ), */
```

Comment:
Nit: this commented-out code should probably be deleted instead?

Reply:
Removed!